src/js/playlist/track.ts

```diff
@@ -6,7 +8,7 @@
  * @property {boolean} default - Enable the track by default.
  */
 
-const Track = function(config) {
+const Track = function(config: PlaylistItemTrack): PlaylistItemTrack | undefined {
```

Comment:
this seems like the file we should type the track in, should i just grab the options from the current config schema and port them? cc @robwalch

Reply:
Should there be a `PlaylistItemTrack` type and a `PlaylistItemTrackOptions` type?

Reply:
i think they are functionally the same in this context right?

Reply:
The same if the optional properties are. Maybe Track always assigns a value to certain props?